crates/fuel-streams-core/src/transactions/mod.rs

```diff
@@ -8,13 +8,11 @@ use types::*;
 use crate::prelude::*;
 
 impl StreamEncoder for Transaction {}
+
 impl Streamable for Transaction {
     const NAME: &'static str = "transactions";
     const WILDCARD_LIST: &'static [&'static str] = &[
```

Comment:
Can we not introduce back the .all() here that would return the list of all Subjects::Wildcards ? ... rather than enumerating them

Reply:
Any reason why? The `.all()` just seems like an extra indirection

Reply:
Just asking, saw @pedronauck had it and you removed. I dont mind the current approach.